passkeys-backend/.env.example

```diff
@@ -0,0 +1,4 @@
+API_URL=
```

Comment:
@nicolas-camacho this is where we need to add descriptions - see [these docs](https://github.com/twilio-labs/configure-env/blob/main/docs/SCHEMA.md) for more info.

Reply:
This is fixed with the comments inside the `.env` file